net-dialup/mgetty/mgetty-1.2.1.ebuild

```diff
@@ -26,7 +26,7 @@ RDEPEND="${DEPEND}
 SLOT="0"
 LICENSE="GPL-2"
 KEYWORDS="alpha amd64 arm hppa ia64 ~mips ppc ppc64 s390 sh sparc x86"
```

Comment:
I asked what's the best approach for KEYWORDS when adding split-usr USE, and got told that they should be reset to ~. So please, make a new revbump and use `ekeyword ~all` on it to reset all KEYWORDS into ~. Also if you'd like to become maintainer for this package, add yourself and proxy-maint to metadata.xml.
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml